pkg/volume/util/util.go

```diff
@@ -82,7 +81,7 @@ const (
 // called 'ready' in the given directory and returns
 // true if that file exists.
 func IsReady(dir string) bool {
-	readyFile := path.Join(dir, readyFileName)
+	readyFile := filepath.Join(dir, readyFileName)
```

Comment:
I don't see unit tests for these functions, could you add some?

Reply:
Added